build/docker/Dockerfile.backend

```diff
@@ -95,3 +95,27 @@ COPY --from=build --chown=node:node /app/dist ./dist
 USER node
 
 CMD yarn start
+
+#####################
+##### NEW STAGE #####
+#####################
+
+# The run stage contains an optimized image for running the application
+FROM run AS migrate
+
+# These parts have to be run as root
+USER root
+
+# yarn db commands require psql
+RUN apk update
+RUN apk add postgresql
```

Comment:
do we want to lock on a specific version of postgres?

Reply:
I don't think it will matter too much given that this is the client and not the server.  The only thing the client is used for is for some DDL commands that set the application up for the real interaction with the server.